src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const books = createSlice({
+  name: '',
+  initialState: [],
```

Comment:
- [ ] Kindly take not that for proper identification, the name is not supposed to be empty. ℹ️ Kindly give it a descriptive name, maybe `books`.